### ISSUE TITLE
DeprecationWarning: Tapable.plugin

### ISSUE THREAD

**Issue by JBowns** [vlazh/node-hot-loader#4]
Hi, I seem to be getting a deprecation warning when using `node-hot-loader` with webpack v4.4.1
```
(node:46156) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
```

After a quick glance at the code a migration might look something like ([HmrServer.js#L221-L222](https://github.com/vlazh/node-hot-loader/blob/master/src/HmrServer.js#L221-L222)):

```javascript
this.context.compiler.hooks.done.tap('CompilerDone', this.compilerDone);
this.context.compiler.hooks.compile.tap('ComplierInvalid', this.compilerInvalid);
```

Thanks

**Comment by vladzdev** [comment]
Hi Jacob!

Thank you very much!
I merged you suggestion in [1.7.0](https://github.com/vlazh/node-hot-loader/blob/master/CHANGELOG.md#170-2018-04-09). Now deprecation warning is gone.

**Comment by JBowns** [comment]
Looking good 👍Thanks
